.storybook/preview-head.html

```diff
@@ -1 +0,0 @@
-<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap" />
```

Comment:
How come this was removed? Roboto is the default font we use in most apps so nice to show it in storybook too if we can.